scripts/sync_version.py

```
@@ -0,0 +1,109 @@
+    init_path.write_text(new_content)
+    print(f"✓ Updated src/aimq/__init__.py to version {version}")
+
+
+def get_current_version() -> str:
+    """Get the current version from pyproject.toml."""
+    pyproject_path = get_project_root() / "pyproject.toml"
+    content = pyproject_path.read_text()
+
+    match = re.search(r'version\s*=\s*"([^"]+)"', content)
+    if not match:
+        raise ValueError("Could not find version in pyproject.toml")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)